CRM/Extendedreport/Form/Report/Price/Lineitem.php

```diff
@@ -84,6 +88,9 @@ public function fromClauses() {
       'address_from_contribution',
       'email_from_contact',
       'phone_from_contact',
+      'first_from_contribution',
+      'Empty_Columns',
```

Comment:
I don't get the 'Empty_Columns', thing?

Reply:
We print out the report to get a tick off list for our CSA logistics. The empty columns are used to get space at the PDF to confirm the receivings by the participant. In my eyes this is the easiest way to influence the rendering of the form.

So yes, the empty columns are pseudofields.

Reply:
Example Report:
![grafik](https://user-images.githubusercontent.com/65115852/85224850-73c24300-b3cd-11ea-810b-0de62d86d501.png)

Reply:
OK - I'll have to set aside some time to think about this - but if this commit https://github.com/eileenmcnaughton/nz.co.fuzion.extendedreport/pull/376/commits/9d6172c6190eef88564d013a2ec6cc9ec43b4c24 

Were in it's own PR I think I could merge that one as soon as it passes tests

Reply:
Thank's. I had to fix the columns to UPPERCASE. The productive SQL server is case sensitive. I did not make an pull request for that.